metals/src/main/scala/scala/meta/internal/metals/Messages.scala

```diff
@@ -188,8 +188,7 @@ object Messages {
       s"${icons.info} Partial navigation",
       tooltip = "This external library source has compile errors. " +
         "To fix this problem, update your build settings to use the same compiler plugins and compiler settings as " +
-        "the external library.",
-      command = ClientCommands.FocusDiagnostics.id
```

Comment:
This is unrelated, but found an issue with focus diagnostics when there were errors in dependencies.